src/lib/tenant/configs/ui/ens.ts

```diff
@@ -132,7 +132,18 @@ export const ensTenantUIConfig = new TenantUI({
             isPreSubmission: false,
           },
         ],
-        proposalTypes: [ProposalType?.BASIC, ProposalType?.SOCIAL],
+        proposalTypes: [
+          {
+            type: ProposalType.BASIC,
+            prodAddress: null,
+            testnetAddress: null,
+          },
+          {
+            type: ProposalType.SOCIAL,
+            prodAddress: null,
+            testnetAddress: null,
+          },
+        ],
```

Comment:
basic and social type do not need to define any address -- there is no module to support for basic or social